docs/user-guide/parametric.rst

```diff
@@ -0,0 +1,82 @@
+.. index::
+   single: parametric rule
+   single: rule; parametric
+
+Parametric rules
+================
+
+General
+-------
+A :term:`parametric rule` is a rule that uses a :cvl:`method f` parameter.
+Such a rule will be tested against all possible methods :cvl:`f`. It is possible to
```

Comment:
Should it also mention that f can be in any contract provided in the list of files?

Reply:
I agree, added a few words on that.